README.md

```diff
@@ -15,4 +17,13 @@ Python data structures.
 $ pip install cbor-decoder
 ```
 
+## Run tests
+
+```
+$ pip install -U pytest
```

Comment:
It is more of a convention. `pip` always uses the default python version. Probably it is python2 in your case. Replaced it with `pip3` though, just to make sure everyone uses python3.